README.md

```diff
@@ -1,8 +1,7 @@
-# IBM Cloud Pak for Business Automation 21.0.1 on Certified Kubernetes
+# IBM Cloud Pak for Business Automation 24.0.0 on Certified Kubernetes
 
-Starting with 21.0.1 IF001 , you will no longer find the folders and resources files for installing IBM Cloud Pak for Business Automation in GitHub. To download the files, go to the Container Application Software for Enterprises (CASE) package and extract the contents from the .tar file.
-See https://www.ibm.com/docs/en/cloud-paks/cp-biz-automation/21.0.x?topic=operator-upgrading-by-running-script
+This repository includes folders and resources to help you install the IBM Cloud Pak for Business Automation capabilities. Installation of the capabilities is done with the Cloud Pak operator.

```

Comment:
@lpchitta can you update this `Installation of the capabilities is done with the Cloud Pak operator`  to `The IBM Cloud Pak for Business Automation operator will execute the installation of the capabilities.`

Reply:
Thank you. Done